snmplib/mib.c

```diff
@@ -4279,6 +4291,14 @@ _get_realloc_symbol(const oid * objid, size_t objidlen,
                                                    subtree->label)) {
                     *buf_overflow = 1;
                 }
+		if (NETSNMP_OID_OUTPUT_FULL_AND_NUMERIC == output_format) { /*RHG: 2021.09.08*/
```

Comment:
No Yoda conditions in new code please. From https://en.wikipedia.org/wiki/Yoda_conditions: "Yoda conditions are criticized for compromising readability by increasing the [cognitive load](https://en.wikipedia.org/wiki/Cognitive_load) of reading the code."